docs/md/dev-process.md

```diff
@@ -6,7 +6,55 @@ within the team,
 and encourage ourselves
 to follow in most cases.
 
-## Summary
+## The Swagger page
+
+WFL ships with a swaggger UI that documents all of the
+available endpoints. It's available at path `/swagger`, e.g.
+https://dev-wfl.gotc-dev.broadinstitute.org/swagger
+
+## Development Setup
+
+Clojure development feels very different from Scala and Java
+development. It even differs markedly from development in other
+*dynamic languages* such as Python or Ruby.
+
+Get a demonstration from someone familiar with Clojure
+development before you spend too much time trying to figure
+things out on your own.
+
+Find a local Cursive user for guidance if you like IntelliJ.
+[Rex Wang](mailto:chengche@broadinstitute.org) and
+[Saman Ehsan](mailto:sehsan@broadinstitute.org) know how to use it.
+Cursive licences are available
+[here](https://broadinstitute.atlassian.net/wiki/spaces/DSDE/pages/48234557/Software%2BLicenses%2B-%2BCursive).
+The steps for getting this project set up with very recent versions of IntelliJ
+differ from Cursive's docs:
+
+1. *Outside of IntelliJ*, `clone` the repo and run `boot` at the top-level to
```

Comment:
I'd recommend running prebuild before launching intellij as this sets up all libraries and derived resources and sources. 

```bash
$ make TARGET=prebuild - jN
```